content/_index.html

```diff
@@ -6,13 +6,19 @@
 
 {{< blocks/cover title="Welcome to FSH School" image_anchor="top" height="full" color="orange" >}}
 <div class="mx-auto">
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" target="_blank" href="/FSHOnline/">
+		Interact with FSH <i class="fas fa-fish "></i>
+	</a>
 	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
-		Learn More <i class="fas fa-fish ml-2"></i>
+		Read the Docs <i class="fas fa-book-open ml-2"></i>
 	</a>
-	<a class="btn btn-lg btn-secondary mr-3 mb-4" target="_blank" href="/FSHOnline/">
-		Experiment <i class="fas fa-flask "></i>
+	<a class="btn btn-lg btn-success mr-3 mb-4" target="_blank" href="{{< relref "/docs/gofsh" >}}">
+		Start a Project <i class="fas fa-pencil-alt "></i>
```

Comment:
I think it might confuse more than a few users for `Start a Project` to go to GoFSH.  I actually expected it to go to documentation for `sushi --init`.  I think that `Convert a Project` or `Switch to FSH` would get the intended point across better.

Reply:
Alternately, we could create a new `Start a Project` section in the doc that branches.  E.g.,
* Starting from an existing FHIR project? GoFSH!
* Starting from scratch? Get started with SUSHI init.

Reply:
I agree. I added a "Quick Start" page at the top level.
At the same time, I demoted the legacy docs, and removed the "About" page (which added very little).

Reply:
Added a Quick Start.